### DISCUSSION TITLE
[Messenger] Doctrine transport and max PostgreSQL integer

### DISCUSSION THREAD

**Discussion by antoniovj1** [symfony/symfony#58815] [Q&A]
I'm using Symfony Messenger and PostgreSQL to manage my application queues. I've seen that the message ID is an auto-incrementing integer. 

I know it's a very large number, but what will happen the day it reaches the largest integer supported by the database?

I've done some research but haven't found anything, thank you very much!

**Comment by MatTheCat** [comment]
From https://www.postgresql.org/docs/current/datatype-numeric.html#DATATYPE-INT:

> Attempts to store values outside of the allowed range will result in an error.
